spec/features/taxonomy_generation/content_preview_modal_spec.rb

```diff
@@ -1,6 +1,13 @@
 require "rails_helper"
 
 RSpec.feature "Content Preview Modal", type: :feature, js: true do
+  let!(:me) do
```

Comment:
Love the names 👍 . Have you considered only adding names on the tests where are used?

Reply:
Yeah agreed - will take them out where they're not used.